coul.c

```diff
@@ -1053,11 +1053,11 @@ void report_init(FILE *fp, char *prog) {
     if (rough)
         fprintf(fp, " -h%u", rough);
     if (opt_batch_min >= 0) {
-        fprintf(fp, " -b%u", opt_batch_min);
+        fprintf(fp, " -b%u", opt_batch_min); // requires 'unsigned int' but the argument type is 'signed int'
```

Comment:
It would be reasonable to change this, but note that it cannot actually go wrong since we are guarded by the check "opt_batch_min >= 0".